ano/environment.py

```diff
@@ -91,7 +91,7 @@ class Environment(dict):
     elif platformSystem == 'Windows':
         arduino_user_dir_guesses.insert(0, os.path.expanduser(os.path.join("~", "My Documents", "Arduino")))
 
-    default_board_model = 'uno'
+    default_board_model = 'mega2560'
```

Comment:
Hmm. I don't think this holds true for the older IDEs that Arturo1.0 is stuck with does it?